examples/04_New Models.ipynb

```diff
@@ -835,6 +835,190 @@
     "## Matrix Models"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "At this point in the notebook, we have seen a few different model types. In all of these models, we have used dictionaries to represent the states. For example, in the implementation of `ThrownObject_ST` above, see how `dx` is defined with a StateContainer dictionary. While all models can be constructed in this way, Matrix Models provide an advanced feature that is applicable and useful for some use-cases. \n",
```

Comment:
```suggestion
    "At this point, we have seen a few different model types. In all of these models, we have used dictionaries to represent the states. For example, in the implementation of `ThrownObject_ST` above, see how `dx` is defined with a StateContainer dictionary. While all models can be constructed in this way, Matrix Models provide an advanced feature that is applicable and useful for some use-cases. \n",
```
I want to avoid calling them notebooks since we're going to generate websites and a book from them.